guides/configuration/appearance/wallet.mdx

```diff
@@ -106,4 +106,152 @@ customStyle: {
 
 This specific custom style above translates into a highly customized version of the wallet model, with gradients, major color changes, and feature removals, as shown in the following image.
 
-<Tip>Try out an [interactive demo showcasing custom styles](https://web-demo.particle.network).</Tip>
\ No newline at end of file
+<Tip>Try out an [interactive demo showcasing custom styles](https://web-demo.particle.network).</Tip>
+
+## Customizing the Wallet Entry Point
+
+Additionaly, you can tailor the location in which the embedded wallet interface will open; in doing so, you can create a custom "Open Wallet" button or merely alter the position of the default Particle Network button.
+
+<Note>Find a full Next.js demo in the [Customizing Wallet Position repository](https://github.com/Particle-Network/custom-wallet-entry).</Note>
+
+### Option 1: Alternative Position for Default Button
```

Comment:
Previously it could have been misunderstood that the only option for positional changes was via. removing the button and handling it through openWallet; although some developers may just want to change where the button is by default, so this section was added to position this part of the document moreso as 3 different ways of moving the wallet entry position.

Reply:
Yes, this is a good addition. But it doesn't work for me like this. 

Using `entryPosition: 'top-left',`  get ` Type '"top-left"' is not assignable to type 'EntryPosition'.

So I tried with the `EntryPosition` object from the wallet package; I got no errors, but the button did not move.

```tsx
import { EntryPosition } from '@particle-network/wallet';

entryPosition: EntryPosition.TR,
```